tests/unit/api_network_test.py

```diff
@@ -104,6 +104,10 @@ def test_create_network(self):
                     }
                 })
 
+    @requires_api_version('1.24')
+    def test_create_network_with_attachable(self):
+        pass
```

Comment:
Sorry I added this empty test stub. I think we can remove it now that there are other tests.